src/stratis_cli/_actions/_physical.py

```diff
@@ -39,8 +55,35 @@ def list_pool(namespace):
         """
         List devices in a pool.
         """
-        # pylint: disable=unused-argument
-        raise StratisCliUnimplementedError("No ability to list pools.")
+        proxy = get_object(TOP_OBJECT)
+        managed_objects = ObjectManager.Methods.GetManagedObjects(proxy, {})
+        (parent_pool_object_path, _) = pools(
+           managed_objects,
+           props={'Name': namespace.pool_name},
+           unique=True
+        )
+        modevs = [MODev(info) for _, info in devs(
+            managed_objects,
+            props={"Pool": parent_pool_object_path},
+        )]
+        tables = [
+            [
+                modev.Devnode(),
+                str(Range(modev.TotalPhysicalSize(), SECTOR_SIZE)),
+                state_val_to_string(modev.State()),
```

Comment:
```
try:
  tables = [
    [ ...,
      ...,
      STATES[modev.State()]
    ] for modev in modevs
  ]
except IndexError as err:
  raise StratisCliError("horrors, stratis cli astonished by stratisd value, try updating to a new version of stratis-cli and./or reporting a bug") from err
```

To encourage the user to update their version of stratis so it works w/ the daemon again and if there is no new version of stratis-cli to treat this as a bug.

Reply:
I don't think that's the right approach for us, since we're systems software. We need to degrade gracefully. Since blockdev state is an informational field only, an out-of-range value is harmless, and should not preclude the user being able to list the blockdevs that make up the pool. We cannot assume that users are always in a position where they are able to do an update or file a bug report. (e.g. maybe on a plane?) but they still may need to use our software for some reason.

Reply:
An out-of-range value for this bit of information is just a symptom of the fact that the CLI is _ getting information from the daemon that it does not understand_. Bad information leads to bad decision making, which can go anywhere. The user should be alerted to the problem. What we're withholding by raising an exception could be some very misleading data. We should not downplay this fact. However, I agree that an exception might be a bit aggressive. How about:

```
STATES[[modev.State()] if modev.State() < STATES.len() else <STANDARD FORMULA TO INDICATE THAT THE RESULT FROM THE DAEMON WAS UNRECOGNIZED>
```

The STANDARD FORMULA can be documented in the CLI manual, so a user would recognize what it means wherever they encounter it.

Reply:
Ok, that works. What were you thinking? Something like `???` or `Unrecognized value`?

Reply:
```???``` is more unsettling, so probably better.